dataminer/About_DataMiner/DataMiner_compliance_offering.md

```diff
@@ -10,7 +10,7 @@ description: Skyline Communications takes pride in the security of its products
 Skyline Communications takes pride in the security of its products and wants to provide the necessary guarantees to its users. For this purpose, among others, an annual security assessment of DataMiner is carried out by a third-party security specialist. In the most recent assessment in 2024, DataMiner's security posture was ranked as above average compared to other applications the security specialist assessed in recent years.
```

Comment:
Possibly we could update this sentence like this to take into account the consistently good results of the past years?
```suggestion
Skyline Communications takes pride in the security of its products and wants to provide the necessary guarantees to its users. For this purpose, among others, an annual security assessment of DataMiner is carried out by a third-party security specialist. In these assessments, DataMiner's security posture has consistently been ranked as above average compared to other applications the security specialist assessed in recent years.
```